tests/python/unittest/test_tvmscript_builder.py

```diff
@@ -23,8 +23,18 @@
 def test_builder_basic():
```

Comment:
move this file to `tests/python/tvmscript` so that our downstream development is all well-tracked.